examples/rpl_udp/udp.c

```diff
@@ -122,7 +122,7 @@ void udp_send(int argc, char **argv)
     memset(&sa, 0, sizeof(sa));
 
     if (address) {
-        ipv6_addr_init(&ipaddr, 0xabcd, 0x0, 0x0, 0x0, 0x0, 0x00ff, 0xfe00, (uint16_t)address);
+        ipv6_addr_init(&ipaddr, 0xabcd, 0x0, 0x0, 0x0, 0x0200, 0x00ff, 0xfe00, (uint16_t)address);
```

Comment:
I don't think abcd is representing the pan id here. The device is configured with an ip address prefixed with `abcd` in rpl_udp/rpl.c . that's why the destination address also has to start with `abcd`